### ISSUE TITLE
downgrade_component cannot emit event to AngularJS in template.

### ISSUE THREAD

**Issue by Jacksunwei** [angular/angular#16722]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

ng1_template.ng
```
<ng2-component on-my-event="ctrl.doSth($event)"></ng2-component>
```
After ClosureCompiler, ctrl.doSth won't get the $event emitted by the Angular component.

**Expected behavior**

ctrl.doSth should be called with the emitted event object.

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 2.0.X

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by Jacksunwei** [comment]
I created a pull request for this. #16724

**Comment by gkalpak** [comment]
Fixed with #16724.
